hollow/src/main/java/com/netflix/hollow/api/consumer/fs/HollowFilesystemBlobRetriever.java

```diff
@@ -138,7 +202,7 @@ public HollowFilesystemBlobRetriever(Path blobStorePath, HollowConsumer.BlobRetr
                 String filename = path.getFileName().toString();
                 if(filename.startsWith("delta-" + currentVersion)) {
```

Comment:
this won't match on `"delta_"+part+"-"+currentVersion+"-"+destinationVersion` is that intentional?

Reply:
Yes, intentional, we only want to match the main blob part here.